tests/mir-opt/pre-codegen/deref_nested_borrows.rs

```diff
@@ -0,0 +1,24 @@
+// EMIT_MIR_FOR_EACH_PANIC_STRATEGY
+
+fn src(x: &&u8) -> bool {
+    // CHECK-LABEL: fn src(
+    // CHECK-NOT: _0 = const true;
+    // CHECK: _0 = Eq({{.*}}, {{.*}});
+    // CHECK-NOT: _0 = const true;
+    let y = **x;
+    unsafe { unknown() };
```

Comment:
Here~ @RalfJung

Reply:
Ah, that's drowning in all the other diff noise for tests changing their output.^^

I find mir-opt diffs completely unreadable so I largely skip them...